src/EditorFeatures/Core/Classification/Semantic/AbstractSemanticOrEmbeddedClassificationViewTaggerProvider.cs

```diff
@@ -156,39 +157,39 @@ protected sealed override ITaggerEventSource CreateEventSource(ITextView textVie
         SnapshotSpan snapshotSpan,
         IClassificationService classificationService,
         ClassificationOptions options,
+        VersionStamp currentSemanticVersion,
         CancellationToken cancellationToken)
     {
-        var range = context.TextChangeRange;
-        if (range == null)
-        {
-            // There was no text change range, we can't just reclassify a member body.
-            return false;
-        }
```

Comment:
instead of having this range be tracked by the context, and passed in.  we can fairly trivially compute it if we store the previous snapshot we computed against, and determine the change ranges between it and the current snapshot we're classifying.